packages/client/mutations/AcceptTeamInvitationMutation.ts

```diff
@@ -175,10 +177,6 @@ const AcceptTeamInvitationMutation: StandardMutation<
         onCompleted(data, errors)
       }
       const {acceptTeamInvitation} = data
-      const isOK = ignoreApproval
-        ? true
-        : handleAcceptTeamInvitationErrors(atmosphere, acceptTeamInvitation)
-      if (!isOK) return
```

Comment:
The purpose of `ignoreApproval` as explained [here](https://github.com/ParabolInc/parabol/pull/6487/files#r866228637) was:

> onError already propagates the error to the form. passing in ignoreApproval prevents a snackbar with the same message from popping up

However, `ignoreApproval` is only used in `EmailPasswordAuthForm` and if a user has already been accepted into a team, `acceptTeamInvitation` will contain an `ALREADY_ACCEPTED` error message. This means that `isOK` will be false and we'll return here. 

By moving this logic below the error message check on line 185, we should be able to prevent the same error message from popping up twice while fixing this bug.